_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Black Duck 2014 Open Source Rookie
+
+[Black Duck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
```

Comment:
I did not intend to word it like that. I wrote it one way then changed it and accidentally a word.